server/index.js

```diff
@@ -28,10 +38,34 @@ app.set('views', 'server/views/');
 // serve static files from /public folder
 const pathToPublicFolder = path.resolve(__dirname, '../public');
 app.use('/public', express.static(pathToPublicFolder));
+ 
+
+app.use(cookieParser());
```

Comment:
Slobodno dodas tu i tamo komentare oblika 'Add module for parsing cookies and request body' ili slicno